docs/configuration.md

```diff
@@ -699,6 +699,48 @@ Advanced options example:
 
 You can define a `customization:` section in the config to configure some aspects of the look and feel of Turnilo.
 
+You can define custom share options section (`shareOptions:`), which will force specific formatting on exported files. It will ovewrite two default CSV and TSV exports in the share menu. You can define specific properties for number, time and time range types in the `locale:` section. Locale and Locale Options sections are optional but they will be set to default values if not provided.
+
+```yaml
+  shareOptions:
+
+  - title: "excel friendly export to TSV" # name displayed in the share menu
+    format: # "tsv" | "csv"
+    separator: # "\t" | "," | ... 
+    lineBreak: "\r\n"
+    finalLineBreak: # "include" | "suppress"
+    columnOrdering: # "keys-first" | "as-seen";
+    locale:
```

Comment:
As Adrian mentioned, there is open issue #504. And again, export should always preserve what is presented on the screen. It applies also to number and date formats. If some formats are not currently available on UI we should focus to  remove existing limitations and add more formatting options. 

Turnilo should be easy to understand for end user. How to

Reply:
I've replied in the #504 . :+1: 
Thank you for review, I will apply and push requested changes soon.
Regarding exports reflecting what's presented on screen/app - please note that this is already not the case. Turnilo is modifying date format on export https://github.com/allegro/turnilo/blob/8824d86d37e354baea45cff492e7e57154daab5d/src/client/utils/tabular-options/tabular-options.ts#L40 so we do get different values.

Consider this export made on latest master - data is in a different format and value is rounded up in the app view.
![image](https://user-images.githubusercontent.com/49651576/70716024-c7035c80-1ceb-11ea-8ae2-d87c35e1a0a3.png)

Comma used as decimal separator is our main business issue - Turnilo uses dot to separate fractional part of a number which causes issues when opened in Excel in our locale (pl-PL).

Reply:
Yep, date & number formats are fixed in CSV/TSV. Currently there is no way to configure format for date dimensions in Turnilo, so dates on UI are formatted in the most concise way. We also put some effort (#288) to keep date formatting consistent across whole app, the export is the only exception with ISO formatting.

On the other hand for metrics, administrator is able to configure default format and user is able to change the configured format on UI, but the formatting is totally ignored by export CSV/TSV :(

Right now I don't know how to design date/number formatting in consistent way. I feel convinced that formatting on UI and formatting in the exports could be different but the formatting is rather related to the given dimension/measure not to particular export. 

Perhaps we should enhance existing formatting configuration to datetime and numeric dimensions in similar way to measures. But for sure start with good design before implementation step.